src/algebraic_geometry/presheafed_space.lean

```diff
@@ -217,6 +217,26 @@ def restrict_top_iso (X : PresheafedSpace C) :
     { erw [op_obj, id_base, opens.map_id_obj], refl },
     { refine has_hom.hom.unop_inj _, exact subsingleton.elim _ _ } } }
 
+/--
+The global sections, notated Gamma.
+-/
+@[simps]
+def Γ : (PresheafedSpace C)ᵒᵖ ⥤ C :=
+{ obj := λ X, (unop X).presheaf.obj (op ⊤),
+  map := λ X Y f, f.unop.c.app (op ⊤) ≫ (unop Y).presheaf.map
+    (hom_of_le $ λ _ _, trivial : ⊤ ⟶ (opens.map f.unop.base).obj ⊤).op,
+  map_id' := λ X, by { op_induction X, erw [unop_id_op, id_c_app, eq_to_hom_refl, id_comp],
+    exact X.presheaf.map_id _, },
+  map_comp' := λ X Y Z f g, by { rw [unop_comp, comp_c_app], simp_rw category.assoc,
+    erw [nat_trans.naturality_assoc, presheaf.pushforward.comp_inv_app, id_comp,
+        category_theory.functor.comp_map, ← map_comp], refl }, }
```

Comment:
multiline `by` is easier to read as a `begin ... end` with newlines!

Reply:
Done.